pkg/server/server.go

```diff
@@ -25,7 +25,8 @@ import (
 type ExperimentalFeatureFlag string
 
 const (
-	AuthorizationModelIDHeader = "openfga-authorization-model-id"
+	AuthorizationModelIDHeader   = "openfga-authorization-model-id"
+	AuthorizationModelIDTraceTag = "authorization-model-id"
```

Comment:
According to https://opentelemetry.io/docs/reference/specification/common/attribute-naming/, keys should use snake_case. 

> For each multi-word dot-delimited component of the attribute name separate the words by underscores (i.e. use snake_case). For example http.status_code denotes the status code in the http namespace.

Could you update this key, please?

Reply:
Done

Reply:
I think you updated the header, and not the trace key, by accident.

Reply:
changed